examples/verify_tsqrtP_calibration.py

```diff
@@ -118,6 +112,14 @@ def indiv_plots(d, thislabel, thiscolor):
             # slightly expand int range to include rising edges
             int_range[0] -= 2e-6
             int_range[-1] += 2e-6
+            # {{{ plot the data that we're integrating all together
+            # PR -- this is the correct choice based on what I'm saying in the PR comments
+            fl.push_marker()
+            fl.basename = None
+            fl.next("collect filtered analytic")
+            fl.plot(abs(s["t":int_range]), alpha=0.3)
+            fl.pop_marker()
+            # }}}
```

Comment:
here I fixed the issues with the diagnostic plot -- note that there is no copy.
I chose to set basename to None to just put everything on the same plot.  The push/pop marker are not the only way of doing this -- if you know the name of the next plot after this, you can just put a "next" after this, instead (or, more a next statement above down, if that makes sense)

Reply:
I think this section ended up removing [this guy](https://github.com/jmfrancklab/proc_scripts/pull/113#discussion_r1733182045) which makes sense so that the different amplitudes aren't all on top of eachother